src/releng_docs/services_shipit.rst

```diff
@@ -56,4 +56,61 @@ It does the following tasks on every run:
 
 
+``shipit v1 and v2 interaction``
+--------------------------------
+
+Initially, shipit-v2 will compliment v1 by adding features, as opposed to replacing. The following is a implementation strategy for how that will work.
+
+**ship-it v1 will solely be responsible for submitting and kicking off a new release**
+
+.. code-block:: bash
+
+    % curl -X POST -d "product=firefox&version=99.0" https://ship-it.mozilla.org/submit_release.html
+
+releaserunner_ continues to behave as normal by polling shipit v1 for new releases and, when found, runs release_sanity_, creates taskcluster_graph1_, and notifies release-drivers of new release.
+In addition, releaserunner will tell shipit-v2 about the new release and ask shipit-v2 to create a pipeline:
+
+.. code-block:: python
+
+    r = requests.post("https://mozilla-releng.net/shipit_pipeline", data={uid='foo', pipeline={}})
+
+shipit-v2's pipeline will consist of steps representing the release. A step could be sign off step (shipit_signoff), or a taskcluster step (shipit_taskcluster).
+
+shipit-v2 will also create a step for the shipit-v1 generated taskcluster graph (graph1) so that it can add that graph to the shipit-v2 pipeline as a dependency. This means that a taskcluster step can be passed an existing taskcluster graphid to track
+rather than always creating a new one.
+
+**How shipit-v1 taskcluster graph1 will differ**
+
+Since graph1 is created by shipit-v1 and includes taskcluster tasks for doing sign offs and publishing releases, it will need to be trimmed and then offload its later tasks to shipit-v2 via subsequent steps within the pipeline.
+
+And so, initially, graph1 will create all required tasks up until the first human sign off. This is easily defined by: 'the first "human decision task" within the graph'.
+
+For example, here is an overview of what a Beta release comprises of in simplified form:
+
+**Betas**
+
+.. image:: img/beta_overview.jpg
+
+In the beta case, graph1 would finish with 'push artifacts to release location'. There would then be a shipit-v2 pipeline consisting of a sign off step: 'publish release human sign off', and a taskcluster step: 'publish release'
+
+The taskcluster 'publish release' step would comprise of the following tasks: publishing on balrog, bumping next version, updating bouncer, and informing shipit-v1 the release is complete (mark as shipped).
+
+Release candidates would be implemented in a similar manner but since it contains more sign offs, shipit-v1 would offload more tasks to shipit-v2
+
+
+``shipit authentication and authorization``
+-------------------------------------------
+
+shipit-v2, at least on initial design, will adopt Taskcluster auth services for both authentication and authorization
+
+This means that rather than re-inventing the wheel or tracking user permissions, shipit-v2 will rely on Taskcluster for its client_, roles_, and scopes model when interacting with any shipit_* api and ui.
```

Comment:
given our recent auth0 and tc auth discussions, I'm going to defer defining this for now. As temp docs, I could put the three options we finalized on, if you haven't written that down somewhere yet? @garbas @mozbhearsum ?